HPCC-16998 Hidl generated warning message needs terminating \n

rpastrana opened the pull request:
- Warning message now contains terminating \n which prevents uninteded logic changes

Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexis.com>

rpastrana commented:
@wangkx please review.

HPCCSmoketest commented:
Automated Smoketest
Sha: 8466a2692e30b362cb4a049bbd9da15282bae48b
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 91.005 seconds |

Install hpccsystems-platform-community_6.2.6-rc1.el7.x86_64.rpm
HPCC Start: OK

Unittest result:

| total | passed | failed | timeout | 
|---|---|---|---|
|88 |88 |0 |0 |


WUtools -selftest result:

| total | passed | failed | timeout | 
|---|---|---|---|
|19 |19 |0 |0 |

HPCC Stop: OK
HPCC Uninstall: OK

wangkx commented:
@rpastrana the change looks fine.

rpastrana commented:
@richardkchapman ready to be merged.

richardkchapman commented:
@rpastrana There's a typo in the commit message

rpastrana committed:
HPCC-16998 HIDL generated warning message needs new line char

- Warning message now contains terminating \n which prevents uninteded logic changes

Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexis.com>

rpastrana commented:
@richardkchapman ready to merge

HPCCSmoketest commented:
Automated Smoketest
Sha: 8e533389495c9545391df88b6a57ba71e7898d0d
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 91.932 seconds |

Install hpccsystems-platform-community_6.2.6-rc1.el7.x86_64.rpm
HPCC Start: OK

Unittest result:

| total | passed | failed | timeout | 
|---|---|---|---|
|88 |88 |0 |0 |


WUtools -selftest result:

| total | passed | failed | timeout | 
|---|---|---|---|
|19 |19 |0 |0 |

HPCC Stop: OK
HPCC Uninstall: OK
